project-koku/koku
# raise exception after logging in execute_delete_sql

Fixes [COST-2710](https://issues.redhat.com/browse/COST-2710)

Changes proposed in this PR:
* re-raise exception after logging in `execute_delete_sql`

Testing Instructions:
1. checkout branch
2. `make docker-reinitdb-with-sources`
3. `python ./koku/manage.py shell`
4. Execute:
    ```
    from koku.database import get_model
    from koku.database import execute_delete_sql
    Customer = get_model("Customer")
    Provider = get_model("Provider")
    cq = Customer.objects.all()
    pq = Provider.objects.filter(uuid=Provider.objects.first().uuid)
    execute_delete_sql(pq)  # no exception
    execute_delete_sql(cq)  # exception thrown after error log
    from django.db import connection
    from koku.database import cascade_delete
    with connection.cursor() as cur:
        cur.execute("drop table api_provider cascade;")

    cascade_delete(Customer, cq)  # Should see warning logs saying Provider table does not exist; but NO EXCEPTIONS
    ```
